google_amoeba_net_d_quater_tpupod_tensorflow18.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "v1.0",
+  "author": "Google",
+  "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+  "framework": "TensorFlow 1.8.0-rc1",
+  "codeURL": "https://github.com/tensorflow/tpu/tree/e5c126d66aa3d25e0cb066bdf7fc46f98fe59901/models/experimental/amoeba_net/",
+  "model": "AmoebaNet-D N6F256",
+  "hardware": "1/4 of a TPUv2 Pod",
+  "timestamp": "2018-04-20",
+  "misc": {
+    "optimizer": "RMSProp",
+    "movingAverageDecay": 0.9999,
+    "weightDecay": 0.00004,
+    "batchSize": 4096,
```

Comment:
Should the batch size for 1/4 of TPUv2 Pod be the same as the batch size for 1/16 of a TPU Pod? Looking at the [previous AmoebaNet-D submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/77f9503ad2a4ee96e652efcef1cb930f6887d730/ImageNet/train/google_amoeba_net_d_tpu_tensorflow18.json#L15) you used a batch size of 1024. I imagine you would have to scale the batch size as the hardware increases, but I don't see that here between 1/16 to 1/4.

Reply:
Nope, for both 1/16 and 1/4 tpu pod the total batch size is 4096.

Reply:
Thanks for confirming that this was intentional. I am surprised that the maximum batch size is lower for this architecture in comparison to ResNet.